CITATION.cff

```diff
@@ -0,0 +1,11 @@
+cff-version: 1.2.0
+message: "If Iris played an important part in your research then please add us to your reference list by using the references below."
+title: "Iris"
+authors:
+    - name: "Iris contributors"
+abstract: "A powerful, format-agnostic, and community-driven Python package for analysing and visualising Earth science data"
+license: "LGPL-3.0"
+doi: "10.5281/zenodo.595182"
```

Comment:
If you agree with my above comment about using the [`version`](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#version) key, then this should become version specific. E.g. `8305232` for `v3.7.0`.

Reply:
https://github.com/SciTools/iris/pull/5483#discussion_r1435115045